samples/Makefile

```diff
@@ -12,14 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SRCS = $(wildcard *.ixtmpl)
-DOCS = $(SRCS:%.ixtmpl=%)
+SRC_DIRS = $(wildcard src/*)
+DIRS_CMD = $(foreach subdir, $(SRC_DIRS), make-rule/$(subdir))
 
+make-rule/%: fetch-files
+	cd $* && $(MAKE) -f ../../Makefile-src viewer
 
-all: $(DOCS:%=%-viewer.html)
+# fetch-sample-files may create new directories, so reinvoke make to get
+# SRC_DIRS re-evaluated.
+all: 
+	./fetch-sample-files.py
```

Comment:
(Arbitrary line.)

The files are already part of the repository history, so removing them only helps shallow clone.  Well, I guess removing them will dissuade people from adding new ones.